modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -40,6 +40,18 @@ image::ex2-exchange-assets.png[]
 
 == Asset Limits
 
+An asset's version format must use semantic versioning. Asset fields have these length limits:
+
+[%header,cols="2a,1a,1a"]
+|===
+|Field |Maximum Length |Minimum Length
+|Name |256 |1
+|`assetId` |256 |1
```

Comment:
```suggestion
```
I insist on removing this length field 😄

Reply:
Done. Thank you!